src/UserSettingsPage.cpp

```diff
@@ -2127,11 +2127,30 @@ UserSettingsModel::exportSessionKeys()
     if (!ok)
         return;
 
+    auto repeatedPassword = QInputDialog::getText(nullptr,
+                                                  tr("Repeat File Password"),
+                                                  tr("Repeat the passphrase:"),
+                                                  QLineEdit::Password,
+                                                  QLatin1String(""),
+                                                  &ok);
+    if (!ok)
+        return;
+
     if (password.isEmpty()) {
         QMessageBox::warning(nullptr, tr("Error"), tr("The password cannot be empty"));
         return;
     }
 
+    if (repeatedPassword.isEmpty()) {
+        QMessageBox::warning(nullptr, tr("Error"), tr("Please repeat the password"));
+        return;
+    }
```

Comment:
This check is unnecessary, if the password was already tested for being empty and password and repeatedPassword have to be the same. You can remove it.